src/main/sensors/compass.c

```diff
@@ -237,7 +237,7 @@ bool compassDetect(magDev_t *magDev, uint8_t *alignment)
         }
 
         if (lis2mdlDetect(magDev)) {
-#ifdef MAG_LIS3MDL_ALIGN
+#ifdef MAG_LIS2MDL_ALIGN
             *alignment = MAG_LIS2MDL_ALIGN;
 #endif
```

Comment:
These `MAG_X_ALIGN` are not defined anywhere.

@ledvinap IMO they either need to be defined or be removed.

Reply:
We have the LIS2MDL mounted with an orientation, that's how I discovered this typo. If no one is using the MAG_*_ALIGN per sensor definitions I propose we use a generic define similar to GYRO_1_ALIGN (ie MAG_ALIGN).